### PR TITLE
Replaced strtok with strtok_s

### PR DESCRIPTION
**Pull request by akallabeth**
Follow up to #6187

### CODE REVIEW

File: client/X11/xf_keyboard.c [hunk 1]
```diff
@@ -74,7 +74,8 @@ static BOOL xf_keyboard_action_script_init(xfContext* xfc)
 
 	while (fgets(buffer, sizeof(buffer), keyScript) != NULL)
 	{
-		strtok(buffer, "\n");
+		char* context = NULL;
```
Comment (kubistika) [inline client/X11/xf_keyboard.c:77]: Shouldn’t this be outside of the loop?

File: libfreerdp/crypto/tls.c [hunk 2]
```diff
@@ -1198,9 +1198,10 @@ static BOOL is_accepted_fingerprint(CryptoCert cert, const char* CertificateAcce
 		char* cur = strtok_s(copy, ",", &context);
 		while (cur)
 		{
+			char* subcontext = NULL;
```
Comment (kubistika) [inline libfreerdp/crypto/tls.c:1201]: Same

File: client/X11/xf_keyboard.c (hunk 1 above)
Comment (akallabeth) [reply client/X11/xf_keyboard.c:77]: no, `strtok` is never called with a `NULL` argument, so each tokenize is a new context

File: libfreerdp/crypto/tls.c (hunk 2 above)
Comment (akallabeth) [reply libfreerdp/crypto/tls.c:1201]: no, the context must be reset for each new `cur`

**Pull request merged**